src/platform-includes/configuration/integrations/pluggable-integrations/javascript.mdx

```diff
@@ -0,0 +1,85 @@
+These pluggable integrations are snippets of code that augment functionality for specific applications and/or frameworks. We document them so you can see what they do and that they can be enabled.
+
+## How to Enable
+
+Install the `@sentry/integrations` package and provide a new instance with your config to `integrations` option. Include the plugin after the SDK has been loaded.
+
+For example:
+
+<PlatformContent includePath="configuration/enable-pluggable-integrations" />
+
+### ExtraErrorData
+
+_Import name: `Sentry.Integrations.ExtraErrorData`_
+
+This integration extracts all non-native attributes from the Error object and attaches them to the event as the `extra` data. If the error object has a `.toJSON()` method, the `ExtraErrorData` integration will run it to extract additional information.
+
+Available options:
+
+<PlatformContent includePath="configuration/extra-error-data" />
+
+### CaptureConsole
+
+_Import name: `Sentry.Integrations.CaptureConsole`_
+
+This integration captures all `Console API` calls and redirects them to Sentry using the SDK's `captureMessage` or `captureException` call depending on the log level. It then retriggers to preserve default native behavior.
+
+<PlatformContent includePath="configuration/capture-console" />
+
+### Debug
+
+_Import name: `Sentry.Integrations.Debug`_
+
+This integration allows you to inspect the content of the processed event, that will be passed to `beforeSend` and effectively send to the Sentry SDK. It will *always* run as the last integration, no matter when it was registered.
+
+Available options:
+
+<PlatformContent includePath="configuration/debug" />
+
+<PlatformSection notSupported={["javascript.electron"]}>
+
+### Offline
+
+_Import name: `Sentry.Integrations.Offline`_
+
+This integration uses the web browser's [online and offline events](https://developer.mozilla.org/en-US/docs/Web/API/NavigatorOnLine/Online_and_offline_events) to detect when no network connectivity is available. If offline, it saves events to the web browser's client-side storage (typically IndexedDB) then automatically uploads events when network connectivity is restored.
+
+This plugin does not attempt to provide local storage or retries for other scenarios. For example, if the browser has a local area connection but no internet connection, then it may report that it's online, and Sentry's `Offline` plugin will not attempt to save or retry any send failures in this case.
```

Comment:
```suggestion
This plugin does not attempt to provide local storage or retries for other scenarios. For example, if the browser has a local area connection but no internet connection, then it may report that it's online, and Sentry's `Offline` plugin will not attempt to save or retry any send failures in this case:
```